.pre-commit-config.yaml

```diff
@@ -0,0 +1,52 @@
+# If you want use pre-commit, you need to install pre-commit package.
+# $ pip install pre-commit
+# $ pre-commit install
+
+default_language_version:
+  python: python3
+
+repos:
+  - repo: https://github.com/psf/black
+    rev: 24.4.0
+    hooks:
+      - id: black
```

Comment:
Don't we need blackdoc?

Reply:
I'm not sure but the official instruction [here](https://github.com/keewis/blackdoc?tab=readme-ov-file#pre-commit) does not work, so I would like to postpone to introduce blackdoc.